lighthouse-core/audits/accessibility/aria-hidden-focus.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview Ensures `aria-hidden` elements do not contain focusable elements.
+ * See base class in axe-audit.js for audit() implementation.
+ */
+
+const AxeAudit = require('./axe-audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of an accesibility audit that checks if all elements that have an aria-hidden attribute do not contain focusable descendent elements. This title is descriptive of the successful state and is shown to users when no user action is required. */
+  title: '`[aria-hidden="true"]` elements do not contain focusable descendents',
+  /** Title of an accesibility audit that checks if all elements that have an aria-hidden attribute do not contain focusable descendent elements. This title is descriptive of the failing state and is shown to users when there is a failure that needs to be addressed. */
+  failureTitle: '`[aria-hidden="true"]` elements contain focusable descendents',
+  /** Description of a Lighthouse audit that tells the user *why* they should try to pass. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
+  description: 'Focusable descendents within an `[aria-hidden="true"]` element will not be available to assistive technologies like screen readers. [Learn more](https://dequeuniversity.com/rules/axe/3.3/aria-hidden-focus).',
```

Comment:
> this one feels a little weak to me because presumably that's exactly why `aria-hidden` was added to the element in the first place, but I guess that's my issue with the opinionated rule rather than this description so 🤷‍♂

yeah, we should maybe even consider a separate `aria-hidden` group in a11y that's basically "if you're going to use `aria-hidden`, keep it really straightforward and don't try anything tricky with it" :)

For this one it sounds more like "don't make non-decorative stuff hidden, and if it's focusable it's probably not decorative" (in addition to it sounds like it'll still be tab focusable but not readable by a screen reader?)